ara/static/css/ara.css

```diff
@@ -80,4 +80,12 @@ td a {
 
 .list table, .list th, .list td {
     text-align: center;
-}
\ No newline at end of file
+}
+
+.incomplete {
```

Comment:
We could probably use a bootstrap-provided icon here instead, see http://getbootstrap.com/components/#glyphicons-glyphs (for example glyphicon-warning-sign )